tutorial/install_dependencies.md

```diff
@@ -85,6 +85,37 @@ as well and you can skip to the next section.
             conda init
             ```
 
+## Get the tutorial repository materials
+If you cloned the workshop repository, then you already have everything you
+ need to set up the tutorial environment, including the notebooks in the
+  `tutorial` subfolder. You can skip to the next section.  
+If you havn't cloned the repository, then at a minimum you will need the 
+  [conda environment file](https://github.com/scipy-2024-image-analysis/tutorial/raw/main/environment.yml) to set up the environment.
```

Comment:
I don't really follow this sentence. I assume people that want the environment (and thus the env file) will also want the notebooks to run and follow the tutorial?